tools/get_release_contributors.py

```diff
@@ -193,38 +200,43 @@ def print_website_contributors(from_, to):
 
 
 def print_contributors(title, git_log_selector, from_, to):
-    contributors = [u for u in find_contributors(git_log_selector, from_, to) if u and u not in ignore_list]
+    contributors = [u for u in find_contributors(git_log_selector, from_, to) if
+                    u and u not in ignore_list]
     contributors_str = ', '.join(contributors)
-    print('%s: %s' % (title, contributors_str))
+    print(f'{title}: {contributors_str}')
 
 
-def find_contributors(git_log_selector, from_, to):
+def find_contributors(git_log_selector, from_ref, to_ref):
     """
     Uses `git log` with the provided git_log_selector to list all commits
-    starting with git reference `from_` and going until reference `to`.
+    starting with git reference `from_ref` and going until reference `to_ref`.
 
     A `git_log_selector` could just be a path such as '.'.
     `from_` and `to` can be any committish such as a commit hash or a tag.
     """
     contributors = set()
     co_author_keys = set()
-    commits = subprocess.check_output(['git', 'log', '-z', '--show-pulls', '--format=format:%H %an <%ae>%n%b', '%s..%s' % (from_, to), '--'] + git_log_selector)
+    commits = subprocess.check_output(
+        ['git', 'log', '-z', '--show-pulls', '--format=format:%H %an <%ae>%n%b',
+         f'{from_ref}..{to_ref}', '--'] + git_log_selector)
     commits = commits.decode(CHARSET)
     for commit in commits.split('\0'):
         if not commit:
             continue
-        hash, author_key = commit.split('\n', 1)[0].split(' ', 1)
-        contributor = authors.get_login_or_realname(author_key, hash)
+        sha, author_key = commit.split('\n', 1)[0].split(' ', 1)
+        contributor = authors.get_login_or_realname(author_key, sha)
         contributors.add(contributor)
-        co_authors = re.findall('Co-authored-by:\s*(\S.*(<[^ >]+>))\s*(?:$|\n)', commit, re.I)
+        co_authors = re.findall(r'Co-authored-by:\s*(\S.*(<[^ >]+>))\s*(?:$|\n)', commit, re.I)
         for co_author_full, co_author_email in co_authors:
-            logger.debug(f'checking co author {co_author_full}')
+            logger.debug('checking co author %s', co_author_full)
             contributor = authors.get_login_or_realname(co_author_full, None)
             if not contributor or not contributor.startswith('@'):
                 # try to find a previous commit by this mail address
                 # and pass this commit id to get_login_or_realname() to retrieve the
-                # associated handle from the github API.
-                commit = subprocess.check_output(['git', 'log', '--format=%H', '--max-count=1', '--author=%s' % re.escape(co_author_email)]).strip().decode(CHARSET)
+                # associated handle from the GitHub API.
+                email = re.escape(co_author_email)
```

Comment:
email is confusing - especially as it seems to be a co-author email.
```suggestion
                esc_email = re.escape(co_author_email)
```
or 

```suggestion
                esc_co_email = re.escape(co_author_email)
```